timeseries/src/autogluon/timeseries/models/local/statsforecast.py

```diff
@@ -178,6 +246,41 @@ def _get_model_type(self):
         return AutoETS
 
 
+class ETSModel(AutoETSModel):
```

Comment:
Why is `ETSModel` designed with parent class `AutoETSModel`, whereas `ARIMAModel` has as parent class `AbstractStatsForecastModel`?

Reply:
StatsForecast doesn't have a dedicated class for the ETS model but has one for ARIMA. The `ETSModel` class is [deprecated](https://github.com/Nixtla/statsforecast/blob/main/statsforecast/models.py#L828) and it's just a wrapper for AutoETS (similar to our code).

My idea was to provide `ETSModel` as a convenience for users who used the `ETSModel` from statsmodels before.